x-pack/plugins/security_solution/server/lib/siem_migrations/rules/data/rule_migrations_data_client.ts

```diff
@@ -14,7 +14,10 @@ import { RuleMigrationsDataRulesClient } from './rule_migrations_data_rules_clie
 import { RuleMigrationsDataResourcesClient } from './rule_migrations_data_resources_client';
 import type { AdapterId } from './rule_migrations_data_service';
 
-export type CreateRuleMigrationInput = Omit<RuleMigration, '@timestamp' | 'status' | 'created_by'>;
+export type CreateRuleMigrationInput = Omit<
+  RuleMigration,
+  '@timestamp' | 'id' | 'status' | 'created_by'
+>;
```

Comment:
I think these 3 types can be removed (`CreateRuleMigrationInput`, `RuleMigrationDataStats`, `RuleMigrationAllDataStats`). they are all duplicated in the _rules_ data client. I forgot to do it in my previous PR, could you please clean them?

Reply:
done